example/app/data/DemoDataSource/plugins/single_view/simple/blueprints/SingleView.blueprint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "SingleView",
+  "type": "CORE:Blueprint",
+  "extends": [
+    "CORE:Entity"
+  ],
+  "attributes": [
+    {
+      "name": "content",
+      "type": "CORE:BlueprintAttribute",
+      "attributeType": "./SingleContent"
```

Comment:
To reduce number of test files, you could just use the "CORE:Entity" directly. No need for that wrapper type

Reply:
Fair point!